.changes/next-release/feature-Python-88171.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "feature",
```

Comment:
Let's mark this as an enhancement. I do not think this warrants a minor version bump especially because it does not change the underlying interpreter used in the official executables and really only affects if trying to install from GitHub.